code/modules/antagonists/blob/overmind.dm

```diff
@@ -290,8 +290,9 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 		forceMove(NewLoc)
 		return TRUE
 
-/mob/camera/blob/mind_initialize()
+/mob/camera/blob/mind_initialize(already_initialized=FALSE)
 	. = ..()
-	var/datum/antagonist/blob/B = mind.has_antag_datum(/datum/antagonist/blob)
-	if(!B)
+	if(already_initialized) // for some reason, they transformed into a blob...
+		to_chat(src, "<span class='boldwarning'>If you were not an antagonist before you did not become one now. You still retain your retain your original loyalties and mind!</span>")
```

Comment:
we have a config for polymorph warnings in policies.txt, either add one or use the existing one